server/engine/src/test/java/org/eclipse/lsp/cobol/usecases/TestCopybookReplacePatterns.java

```diff
@@ -175,7 +175,7 @@ void testWhenReplacedLengthIsMoreThanReplaceable() {
             "1",
             new Diagnostic(
                 new Range(),
-                "Max length limit of 30 bytes allowed for variable name.",
+                "variable name cannot exceed 30 characters",
```

Comment:
Capital letter "Variable" (repeat on the other two lines)

Reply:
updated